### PR TITLE
[backend] Add a missing basePath (#8205)

### LINKED ISSUE #8205: app.base_path configuration is not working anymore (default empty works fine)
## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: { e.g. OpenCTI 1.0.2 } 6.2.15
3. OpenCTI client: { e.g. frontend or python }
4. Other environment details:

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Set up a app.base_path that is not empty
2. Run OpenCTI, login, navigate, logout

I tested with version 6.2.11 and app.base_path works correctly.

## Expected Output

everything works fine

## Actual Output

- all call to graphQL are in error, platform UI is all broken, I cannot navigate.

### PR DESCRIPTION
**Pull request by aHenryJard**
### Proposed changes

* Make app.base_path fully work.

### Related issues

* closes #8205 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-graphql/src/http/httpPlatform.js [hunk 1]
```diff
@@ -489,10 +489,11 @@ const createApp = async (app) => {
   });
 
   // Error handling
-  app.use((err, req, res) => {
+  app.use((err, req, res, next) => {
     const error = UnknownError('Http call interceptor fail', { cause: err, referer: req.headers.referer });
     logApp.error(error);
     res.status(500).send({ status: 'error', error: err.stack });
+    next(err);
```
Comment (aHenryJard) [inline opencti-platform/opencti-graphql/src/http/httpPlatform.js]: This part is a fix on top of https://github.com/OpenCTI-Platform/opencti/pull/8140

**Comment by lndrtrbn** [comment]
Is there any additional steps than modifying the bae path in default.json ? I can't get the frontend working

**Review by richard-julien — DISMISSED** [review]

**Comment by richard-julien** [comment]
Deepscan need to be fixed but im fine with the PR content

**Comment by aHenryJard** [comment]
> Is there any additional steps than modifying the bae path in default.json ? I can't get the frontend working

For some reason the frontend for development is not correctly configured I didn't had time to fix it yet, I'm testing that fix by building the app (yarn build inside opencti-graphql and then just yarn start and open http://localhost:4000).

**Review by labo-flg — APPROVED** [review]

**Pull request merged**